src/components/header/Header.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import './_Header.scss'
+import imageLogo from '../../assets/compact-disc-solid.png'
```

Comment:
Don't use absolute paths. If you ever change the component location, you'd need to update the import too.
Instead, use module resolvers and create an alias for the `assets` folder

https://parceljs.org/module_resolution.html